_posts/2017-03-27-open source update.md

```diff
@@ -0,0 +1,93 @@
+---
+published: true
+layout: post
+author: Catherine Farman
+title: "History and future of open source at CFPB"
+permalink: /articles/open-source-update/
+---
+
```

Comment:
Potentially out of scope for this PR: the front page of the cfpb.github.io site looks very odd with this PR included. Issues with spacing, and alignment with the top of the content versus the top of the sidebar.

![screen shot 2017-05-22 at 3 48 53 pm](https://cloud.githubusercontent.com/assets/1183435/26325943/05d7f2c4-3f07-11e7-943d-9b352e1af176.png)